docker/wrapper.py

```diff
@@ -1,136 +1,433 @@
+        type : {star_type}
+        index : {star_index}
+    bwa:
+        version: {bwa_ver}
+        index : {bwa_index}
+    post:
+        samtools:
+            version: {samtools_alignment_ver}
+        picard:
+            version: {picard_ver}
+
+expression_estimation:
+    rsem:
+        version: {rsem_ver}
+        index : {rsem_index}
+
+mutation_calling:
+    indexes:
+        chromosomes: {chromosomes}
+        genome_fasta : {genome_fasta}
+        genome_fai : {genome_fai}
+        genome_dict : {genome_dict}
+        cosmic_vcf : {cosmic_vcf}
+        cosmic_idx : {cosmic_idx}
+        dbsnp_vcf : {dbsnp_vcf}
+        dbsnp_idx : {dbsnp_idx}
+        dbsnp_tbi : {dbsnp_tbi}
+    mutect:
+        version: {mutect_ver}
+        java_Xmx : {mutect_javaxmx}
+    muse:
+        version: {muse_ver}
+    radia:
+        version: {radia_ver}
+        cosmic_beds: {cosmic_beds}
+        dbsnp_beds: {dbsnp_beds}
+        retrogene_beds: {retrogene_beds}
+        pseudogene_beds: {pseudogene_beds}
+        gencode_beds: {gencode_beds}
+    somaticsniper:
+        version: {somaticsniper_ver}
+        samtools:
+            version: {samtools_somaticsniper_ver}
+        bam_readcount:
+            version: {bamreadcount_ver}
+    star_fusion:
+        run: {starfusion}
+        version: {star_fusion_ver}
+    fusion_inspector:
+        run_trinity: {run_trinity}
+        version: {fusioninspector_ver}
+    strelka: 
+        version: {strelka_ver}
+        config_file: {strelka_config}
+
+mutation_annotation:
+    snpeff:
+        version: {snpeff_ver}
+        index : {snpeff}
+        java_Xmx : {spnff_javaxmx}
+
+mutation_translation:
+    transgene:
+        version: {transgene_ver}
+        gencode_peptide_fasta : {transgene_pep}
+        gencode_transcript_fasta : {transgene_trans}
+        gencode_annotation_gtf : {transgene_anno}
+        genome_fasta : {transgene_genome}
+
+haplotyping:
+    phlat:
+        version: {phlat_ver}
+        index : {phlat}
+
+mhc_peptide_binding:
+    mhci:
+        version: {mhci_ver}
+        method_file : {mhci}
+        pred : {mhci_pred}
+
+    mhcii:
+        version: {mhcii_ver}
+        method_file : {mhcii}
+        pred : {mhcii_pred}
+    netmhciipan:
+        version: {netmhciipan_ver}
+
+prediction_ranking:
+    rankboost:
+        version: {rankboost_ver}
+        mhci_args: 
+            npa: {mhci_npa}
+            nph: {mhci_nph}
+            nMHC: {mhci_nMHC}
+            TPM: {mhci_TPM}
+            overlap: {mhci_overlap}
+            tndelta: {mhci_tndelta}
+        mhcii_args:
+            npa: {mhcii_npa}
+            nph: {mhcii_nph}
+            nMHC: {mhcii_nMHC}
+            TPM: {mhcii_TPM}
+            tndelta: {mhcii_tndelta}
+
+reports:
+    mhc_pathways_file: {mhc_pathways_file}
+    itx_resistance_file: {itx_resistance_file}
+    immune_resistance_pathways_file: {immune_resistance_pathways_file}
+    car_t_targets_file: {car_t_targets_file}""")
+
+
+def str2bool(v):
```

Comment:
Add a docstring, add the stackoverflow link to the question where you found this answer to the docstring